tar1090/00-startup-checks

```diff
@@ -0,0 +1,51 @@
+#!/command/with-contenv bash
+# this ^ is a custom s6 helper for bringning in env variables
+set -e
+
+# Bring in the common scripts from sdr-enthusiasts base image
+source /scripts/common
+
+# Check if service has been disabled through the DISABLED_SERVICES environment variable.
+
+if [[ ",$(echo -e "${DISABLED_SERVICES}" | tr -d '[:space:]')," = *",$BALENA_SERVICE_NAME,"* ]]; then
+        echo "$BALENA_SERVICE_NAME is manually disabled. Sending request to stop the service:"
+        curl --header "Content-Type:application/json" "$BALENA_SUPERVISOR_ADDRESS/v2/applications/$BALENA_APP_ID/stop-service?apikey=$BALENA_SUPERVISOR_API_KEY" -d '{"serviceName": "'$BALENA_SERVICE_NAME'"}'
```

Comment:
```suggestion
        curl --retry 10 --retry-all-errors --header "Content-Type:application/json" "$BALENA_SUPERVISOR_ADDRESS/v2/applications/$BALENA_APP_ID/stop-service?apikey=$BALENA_SUPERVISOR_API_KEY" -d '{"serviceName": "'$BALENA_SERVICE_NAME'"}'
```

Reply:
ahh yes - good one! missed that from your recent updates 👍